types/webmidi/index.d.ts

```diff
@@ -4,193 +4,234 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
btw, not sure at what point these typings became considered applicable to https://github.com/djipco/webmidi (see line 2 folded above, which I can't directly comment on) but I think that's not the case (see discussion here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/39571). Mind removing it?

Reply:
Done.